components/serializer.rst

```diff
@@ -1254,6 +1247,52 @@ These are the options available:
 ``remove_empty_tags``
     If set to true, removes all empty tags in the generated XML.
 
+The CSV Encoder
+---------------
```

Comment:
IMO we should keep the same underline as XML encoder, because the should be on the same hierachy

Reply:
It already is on the same level, check out the rendered docs: https://pr-12461-6xsc4kq-nq3xa5jtywvyc.eu.s5y.io/components/serializer.html